src/socket.rs

```diff
@@ -534,6 +534,22 @@ impl UtpSocket {
         }
     }
 
+    #[cfg(windows)]
+    fn ignore_udp_error(e: &Error) -> bool {
+        match e.raw_os_error() {
+            Some(e) => match e {
+                10054 | 10040 => true,
```

Comment:
Yes, the commit message is good

> Do you want me to add a comment before the code or something like that?

Yes please, the commit message will be lost sooner or later, so would be nice for a new programmer to know what this is about. Perhaps just using WSAECONNRESET and WSAEMSGSIZE instead of the numbers (can be set as const above the `match`) and adding those links from the commit message would be good enough.